### DISCUSSION TITLE
Insert FieldSchema in milvus v1.1.1

### DISCUSSION THREAD

**Discussion by ngominhhaibk** [milvus-io/milvus#27742] [Q&A and General discussion]
I can only insert vectors into collection. Can I insert FieldSchema in milvus v1.1.1, such as name, age, time, ...

**Comment by yhmo** [comment]
No, milvus v1.x only has two built-in fields: id and vector. It doesn't support scalar fields(name, age, time, ...). Usually, users of milvus v1.x store vectors in milvus, and store scalar fields in MySQL/Postgres. They search a vector and get the result ids from milvus, then use the ids to query scalar values from MySQL/Postgres.
